bundle/src/main/java/com/adobe/acs/commons/wcm/comparisons/lines/Line.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *  * #%L
+ *  * ACS AEM Commons Bundle
+ *  * %%
+ *  * Copyright (C) 2016 Adobe
+ *  * %%
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *      http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *  * #L%
+ *
+ */
+package com.adobe.acs.commons.wcm.comparisons.lines;
+
+import com.google.common.base.Optional;
```

Comment:
This is actually a bit tricky. We embed Guava and it can't be part of the API of ACS AEM Commons since our embedded version and the version provided by AEM are not the same. I don't 100% remember why we embed it; I assume it is for cross-version compatibility.

I would expect that this produces a warning from the maven-bundle-plugin about using a non-exported class in an exported package (or something like that). That needs to be fixed.